pkg/asset/kubeconfig/admin.go

```diff
@@ -34,11 +40,20 @@ func (k *Admin) Generate(parents asset.Parents) error {
 		adminCertKey,
 		installConfig.Config,
 		"admin",
-		"kubeconfig",
+		kubeconfigAdminPath,
 	)
 }
 
 // Name returns the human-friendly name of the asset.
 func (k *Admin) Name() string {
 	return "Kubeconfig Admin"
 }
+
+// Load returns the kubeconfig from disk.
+func (k *Admin) Load(p asset.FilePatternFetcher) (found bool, err error) {
```

Comment:
The kubeconfig files are not written to disk by the installer, so they will never by loaded.

Reply:
The admin kubeconfig will be written by the cluster target, right?

Reply:
Ah, yes. You are correct.